### PR TITLE
#8Task. Ihar Tsykala

### PR DESCRIPTION
**Pull request by IharTsykala**
https://travis-ci.org/github/IharTsykala/js-assignments/builds/711798462

### CODE REVIEW

File: task/08-objects-tasks.js [hunk 1]
```diff
@@ -107,40 +108,135 @@ function fromJSON(proto, json) {
...
-    },
-
-    combine: function(selector1, combinator, selector2) {
-        throw new Error('Not implemented');
-    },
-};
-
+class Selector {
+  constructor() {
+    this.index_of_selector = 0
```
Comment (KostyaKov) [inline task/08-objects-tasks.js]: You should put ";" at the end of variable and expression declarations

**Review by KostyaKov — COMMENTED** [review]
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by IharTsykala** [comment]
Please, accept my pr

**Comment by KostyaKov** [comment]
Please add and push fixes to the task according to the comments.
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by IharTsykala** [comment]
I excluded changes in exist code and comment. Please, accept my pr

https://travis-ci.org/github/IharTsykala/js-assignments/builds/713252611

**Comment by IharTsykala** [comment]
I excluded changes in exist code and comment. Please, accept my pr

https://travis-ci.org/github/IharTsykala/js-assignments/builds/713985356

**Comment by IlyaMokin** [comment]
Looks good, I am closing the PR

**Pull request closed without merging**